src/engine-actions/get-status.js

```diff
@@ -21,6 +21,20 @@ const {
  */
 const WALLET_EXISTS_ERROR_MESSAGE = 'wallet already exists'
 
+function parseVersion (version) {
```

Comment:
does this handle (and does it need to handle) versions that start with `v`

Reply:
It doesn't handle that and lnd doesn't currently return a `v` so shouldn't be necessary.